### PR TITLE
(maint) Update acceptance testing docs

### PR DESCRIPTION
**Pull request by mwaggett**
This commit updates the acceptance README to be prescriptive about how to run
beaker tests locally. This also removes old documentation and host configs that
are no longer necessary when using the handy shell script to test.

### CODE REVIEW

File: acceptance/config/beaker/jenkins/el-6-x86_64-mdca-a.cfg [hunk 1]
```diff
@@ -1,25 +1,21 @@
 HOSTS:
-  master:
+  primary:
```
Comment (justinstoller) [inline acceptance/config/beaker/jenkins/el-6-x86_64-mdca-a.cfg:2]: nice

Comment (mwaggett) [reply acceptance/config/beaker/jenkins/el-6-x86_64-mdca-a.cfg:2]: I assumed that these were arbitrary names and thus changeable, as opposed to the actual roles we set below.

Comment (tvpartytonight) [reply acceptance/config/beaker/jenkins/el-6-x86_64-mdca-a.cfg:2]: Do we actually use these anywhere? I assumed everything is a BHG string for our ci...

**Comment by justinstoller** [comment]
fwiw, I use `acceptance/scripts/generic/testrun.sh` when I run locally and that use beaker hostgenerator. We may want to consolidate how we run these tests locally.

**Comment by Magisus** [comment]
I also use the `testrun` script. I recently(ish) updated it to use beaker subcommands.

**Comment by mwaggett** [comment]
I don't think Jenkins does, since they were still using vcloud. I was using these when I was reproducing the ring failures.
If there are other ways to run tests, we should really update our developer docs. I also struggled to figure out how to test things via the repl.

**Comment by Magisus** [comment]
There is a really long doc about acceptance testing. The script Justin and I are talking about it is described here https://github.com/puppetlabs/puppetserver/tree/6.x/acceptance#using-the-handy-shell-script

**Comment by mwaggett** [comment]
oh right, I did read that, but I wanted to run specific tests. I didn't actually look at the test script to see that you could override the suites. I thought it just ran 'em all or something.

**Comment by Magisus** [comment]
No it's really customizable.

**Comment by mwaggett** [comment]
fwiw, I don't find "really long" docs useful. I think we should remove everything about other ways to run the tests if the test script is what's recommended.

I guess I can update this PR and remove these configs altogether if we think that's the right direction.

**Comment by Magisus** [comment]
It looks like Jenkins uses a [generic script](https://github.com/puppetlabs/ci-job-configs/blob/master/resources/scripts/integration-beaker.sh) that leverages many of these same env vars and uses beaker subcommands.

**Comment by mwaggett** [comment]
okay let me know what y'all think of this update

**Review by Magisus — APPROVED** [review]
I think this looks good! Thanks for cleaning this up.

**Review by Magisus — APPROVED** [review]

**Pull request merged**